ChocolateyProvider.csproj

```diff
@@ -64,7 +64,7 @@
   </ItemGroup>
   <ItemGroup>
     <Reference Include="chocolatey, Version=0.9.10.0, Culture=neutral, PublicKeyToken=79d02ea9cad655eb, processorArchitecture=MSIL">
-      <HintPath>packages\chocolatey.lib.0.9.10-beta-20160411\lib\chocolatey.dll</HintPath>
+      <HintPath>packages\chocolatey.lib.0.9.10-beta-20160402\lib\chocolatey.dll</HintPath>

```

Comment:
Yes, had a change that required it. I was able to fix up choco.exe prior to approval, but NuGet immediately publishes packages.